### PR TITLE
Implement greenhouse internal pages

### PR DESCRIPTION
**Pull request by natapg**
Closes [WEB-408](https://fpjs.myjetbrains.com/youtrack/issue/WEB-408)

### CODE REVIEW

File: src/pages/careers/jobs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+  const handleSelectedCategory = (option) => {
+    const department = option.value
+    setSelectedDepartment(department)
+    const filterJobs =
+      department === defaultDepartment
+        ? jobs
+        : jobs?.filter((job) => {
+            return job.department === department
+          })
```
Comment (tan-90) [inline src/pages/careers/jobs/index.tsx:65]: Minor: no need for a function body

File: src/pages/careers/jobs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+          }
+        })
+        const uniqueDepartments = new Set(jobs.map((job) => job.department))
+        const departmentsArray = Array.from(uniqueDepartments)
+
+        setDepartments(departmentsArray)
+        setJobs(jobs)
+        setIsLoading(false)
+      })
+      .catch(() => onError())
```
Comment (tan-90) [inline src/pages/careers/jobs/index.tsx:88]: Minor: I'd change this to be an async function as well.

File: src/pages/careers/jobs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+        </Container>
+        <Container size='large'>
+          <h1 className={styles.title}>Current Job Openings</h1>
+          {hasError ? (
+            <ErrorMessage />
+          ) : (
+            <>
+              <div className={styles.selectSection}>
+                <label className={styles.selectLabel}>Department</label>
+                <Select
```
Comment (tan-90) [inline src/pages/careers/jobs/index.tsx:181]: Minor: I think the default select highlight color doesn't match the style very well.

**Review by tan-90 — CHANGES_REQUESTED** [review]

File: src/pages/careers/jobs/apply/index.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+import { Helmet } from 'react-helmet'
+import { isBrowser } from '../../../../helpers/detector'
+import { PATH } from '../../../../constants/content'
+import styles from './Apply.module.scss'
+import { GREENHOUSE_COMPANY_ID } from '../../../../constants/env'
+import { getJobInfoFromGreenhouse } from '../../../../helpers/api'
+import { Job, ErrorMessage } from '../index'
+import Skeleton from '../../../../components/Skeleton/Skeleton'
+import { decode } from 'html-entities'
+import { scrollToElementById } from '../../../../helpers/scrollToElementBytID'
```
[the remark this answers is not in the document]
Comment (natapg) [reply src/pages/careers/jobs/apply/index.tsx:23]: yeah, you are right! I'll create a ticket

**Review by tan-90 — APPROVED** [review]

**Pull request merged**